randovania/games/dread/json_data/Cataris.txt

```diff
@@ -124,7 +124,7 @@ Extra - asset_id: collision_camera_001
   * Layers: default
   * Slide Tunnel to Underlava Puzzle Room 2/Tunnel to Dropdown Pit
   > Tunnel to Above Z-57 Fight
-      Space Jump or Simple IBJ
+      Simple IBJ or Use Spin Boost
```

Comment:
i reckon you can put "Speed Booster and Can Slide" as an option too. I think the hallway in Underlava is long enough to charge one. Otherwise, any other methods of climbing would be using tricks.

Reply:
We aren't doing multi room tricks with speed are we?